plugins/modules/pacemaker_resource.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2025, Dexter Le <dextersydney2001@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: pacemaker_resource
+short_description: Manage pacemaker resources
+author:
+  - Dexter Le (@munchtoast)
+version_added: 10.4.0
+description:
+  - This module can manage resources to a pacemaker cluster from Ansible using
+    the pacemaker cli.
```

Comment:
More of a style recommendation, although capitalization is usually required.
```suggestion
  - This module can manage resources in a Pacemaker cluster using the pacemaker CLI.
```